Binary files qwt-5.0.2.orig/.DS_Store and qwt-5.0.2/.DS_Store differ
diff -urN qwt-5.0.2.orig/qwtconfig.pri qwt-5.0.2/qwtconfig.pri
--- qwt-5.0.2.orig/qwtconfig.pri	2007-06-11 01:41:29.000000000 -0400
+++ qwt-5.0.2/qwtconfig.pri	2007-09-29 17:39:04.000000000 -0400
@@ -3,7 +3,7 @@
 ######################################################################
 
 unix {
-    INSTALLBASE    = /usr/local/qwt-5.0.2
+    INSTALLBASE    = @PREFIX@/lib/qt4-x11
 }
 
 win32 {
@@ -11,8 +11,13 @@
 }
 
 target.path    = $$INSTALLBASE/lib
-headers.path   = $$INSTALLBASE/include
-doc.path       = $$INSTALLBASE/doc
+headers.path   = $$INSTALLBASE/include/qwt
+doc.path       = @PREFIX@/share/doc/qwt
+doc.files      = doc/html 
+man.path       = @PREFIX@/share/man
+man.files      = doc/man
+
+QMAKE_LFLAGS_SONAME = -install_name$${LITERAL_WHITESPACE}@PREFIX@/lib/qt4-x11/lib/
 
 ######################################################################
 # qmake internal options
@@ -82,4 +87,4 @@
 # Otherwise you have to build them from the examples directory.
 ######################################################################
 
-#CONFIG     += QwtExamples
+CONFIG     += QwtExamples
